_data/es.yml

```diff
@@ -0,0 +1,344 @@
+  iosNotificationApp: Obtenter la app de notificaciones de iOS
+  getStarted: Comenzar »
+  runningWindows: Parece que está ejecutando Windows.
+  runningMac: Parece que está ejecutando Mac
+  runnigDebian: Parece que está usando Debian/Ubuntu
+  runningRedHat: Parece que está usando Red Hat/Fedora
+  currentVersion: Versión actual
+  hFeatures: Características
+  hIdentityVerifications: Sin verificación de identidad
+  pIdentityVerifications: Accesible instantáneamente. No es necesario registrarse o una aprobación desde una autoridad central.
+  hDecentralized: Descentralizado
+  pDecentralized: Los intercambios se llevan a cabo en una red global P2P de usuarios ejecutando Bisq en sus propias máquinas.
+  hSafe: Seguro
+  pSafe: "Bisq nunca retiene sus fondos. Los depósitos de seguridad y los monederos de intercambio multifirma permiten intercambios seguros y exitosos."
+  hPrivate: privado
+  pPrivate: Nadie excepto los pares de intercambio intercambian datos de identificación personal. Todos los datos se guardan de manera local.
+  hSecure: Seguro y siempre en funcionamiento
+  pSecure: Comunicación cifrada de extremo a extremo enrutada vía Tor y sin caídas.
+  hOpen: Abierto
+  pOpen: "El código es <a href=\"https://github.com/bisq-network/bisq\" target=\"_blank\" rel=\"noopener\">abierto</a>, y la estrategia del proyecto se discute abiertamente en <a href=\"https://bisq.network/slack-invite\" target=\"_blank\" rel=\"noopener\">Slack</a> y <a href=\"https://github.com/bisq-network/bisq\" target=\"_blank\" rel=\"noopener\">GitHub</a>."
+  hEasy: Fácil de Usar
+  pEasy: "<p class=\"grey px-md-5\">Hemos hecho toda esta sofisticación simple.<a href='https://docs.bisq.network/getting-started' target='_blank' rel=\"noopener\">Haga su primer intercambio</a> en menos de 10 minutos!</p>"
+  hBisqDao: La DAO Bisq
+  pBisqDao1: Gobernanza descentralizada para un exchange descentralizado.
+  pBisqDao2: Aprenda más aquí
+  hGetStarted: Cómo Empezar
+  divGetStarted: "Con Bisq, puede comenzar a intercambiar en menos de 10 minutos:"
+  liGetStarted1: Descargar la app
+  liGetStarted2: ", o construirla desde la fuente."
+  liGetStarted3: >
+    <li class="mt-0 mb-4 pb-4 border-bottom step step-text-2" data-image="step-image-2">Añada sus cuentas de moneda nacional o cripto</li> <li class="mt-0 mb-4 pb-4 border-bottom step step-text-3" data-image="step-image-3">Cree una nueva oferta o tome una existente.</li> <li class="mt-0 step step-text-4" data-image="step-image-4">Siga la interfaz de usuario para finalizar el intercambio.</li>
+  hJoin: Unirse a la comunidad
+  pJoin: >
+    <p>Bisq siempre está buscando desarrolladores altamente cualificados, diseñadores y expertos en seguridad para unirse a nuestra <a href="localhost:4001/community/">comunidad</a>. Si está interesado en ayudar a mejorar Bisq, hay muchas maneras de<a href="https://docs.bisq.network/contributor-checklist.html" target="_blank" rel="noopener">contribuir</a>.
```

Comment:
I don't know the language but it seems like there are missing spaces around the links here.

Notice "maneras **decontribuir**" and "Nos **encantaríasaber** de ti".

![Screenshot from 2019-08-08 00-28-38](https://user-images.githubusercontent.com/735155/62675205-fc7def00-b973-11e9-81e5-c6d986fafb7d.png)